### PR TITLE
Improve code coverage

### PR DESCRIPTION
**Pull request by guedou**
This PR aims to improve code coverage.

### CODE REVIEW

File: scapy/utils.py [hunk 1]
```diff
@@ -326,23 +326,17 @@ def hexdiff(x, y):
...
+    checksum_endian_transform = lambda chk: ((chk >> 8) & 0xff) | chk << 8
     def checksum(pkt):
         if len(pkt) % 2 == 1:
             pkt += b"\0"
         s = sum(array.array("H", pkt))
         s = (s >> 16) + (s & 0xffff)
         s += s >> 16
         s = ~s
-        return (((s >> 8) & 0xff) | s << 8) & 0xffff
+        return checksum_endian_transform(s) & 0xffff
```
Comment (gpotter2) [inline scapy/utils.py]: You'll have to decrease the indentation of `checksum()` :-)

**Comment by guedou** [comment]
Thanks! Could you have a look at the two new commits?

**Comment by guedou** [comment]
Ready to be merged. The failing test is related to Python 3.4

**Comment by guedou** [comment]
@p-l- @gpotter2 ready to be reviewed the error is related to Pythohn3.4 tests on Travis.

**Comment by guedou** [comment]
@gpotter2 could you re-review and merge?

**Pull request merged**